kafka-admin/.openapi/kafka-admin-rest.yaml

```diff
@@ -792,6 +794,45 @@ paths:
           $ref: '#/components/responses/NotAuthorized'
         "500":
           $ref: '#/components/responses/ServerError'
+  /api/v1/errors:
```

Comment:
Question (I did not run it yet) is that accessible without authentication?

Reply:
Yes, there is no authentication. I'll make sure that's represented in the API definition.